events/bingo/bingoBossApp/bingoBossApp_app.js

```diff
@@ -84,42 +101,53 @@
         }, FAREWELL_SOUND.duration * MS_PER_S * NINETY_PERCENT);
     }
 
+
+    // Tell the number wheel to give prizes to avatars in the prize zones
     function givePrizes() {
         Entities.callEntityServerMethod(NUMBER_WHEEL, 'givePrizes');
     }
 
-    /* ON WEB EVENT: Call the correct function or print an error when an event is received from bingoBossApp_ui.html */
+    // Handle EventBridge Web Events from bingoBossApp_ui.html
     function onWebEventReceived(event) {
-        if (event.app === 'bingo') {
+        if (event.app === 'bingoBossApp') {
             switch (event.type) {
+                case "eventBridgeReady":
+                    sendStatusUpdateToUI();
+                    break;
                 case 'lightsOn':
                     lightsOn();
+                    sendStatusUpdateToUI("Last button clicked: Lights On");
                     break;
                 case 'openRegistration':
                     openRegistration();
+                    sendStatusUpdateToUI("Last button clicked: Open Registration");
                     break;
                 case 'closeRegistration':
                     closeRegistration();
+                    sendStatusUpdateToUI("Last button clicked: Close Registration");
                     break; 
                 case 'newRound':
                     newRound();
+                    sendStatusUpdateToUI("Last button clicked: New Round");
                     break;
                 case 'lightsOut':
                     lightsOut();
+                    sendStatusUpdateToUI("Last button clicked: Lights Out");
                     break;
                 case 'givePrizes':
                     givePrizes();
+                    sendStatusUpdateToUI("Last button clicked: Give Prizes");
                     break;
                 default:
                     print("error in detecting event.type in Bingo app");
             }
         }
     }
 
-    /* ON APP START: Setup app UI, button, and messaging between it's html page and this script */
+    // Setup AppUI module
     var ui;
     var AppUi = Script.require('appUi');
-    var appPage = Script.resolvePath('bingoBossApp_ui.html?5');
+    var appPage = Script.resolvePath('ui/bingoBossApp_ui.html?1');
```

Comment:
Small:  You have }()), but IIFE is })()

Reply:
i'm sorry, i don't follow. what needs to be changed here?

Reply:
https://developer.mozilla.org/en-US/docs/Glossary/IIFE (the bottom of your file)

Reply:
AH! ty :)